docs/source/geodesic.rst

```diff
@@ -1,219 +1,217 @@
+
+If points 1, 2, and 3 lie on a single geodesic, then the following
+addition rules hold:
+
+* *s*\ :sub:`13` = *s*\ :sub:`12` + *s*\ :sub:`23`
+* σ\ :sub:`13` = σ\ :sub:`12` + σ\ :sub:`23`
+* *S*\ :sub:`13` = *S*\ :sub:`12` + *S*\ :sub:`23`
+* *m*\ :sub:`13` = *m*\ :sub:`12`\ *M*\ :sub:`23` +
+  *m*\ :sub:`23`\ *M*\ :sub:`21`
+* *M*\ :sub:`13` = *M*\ :sub:`12`\ *M*\ :sub:`23` −
+  (1 − *M*\ :sub:`12`\ *M*\ :sub:`21`)
+  *m*\ :sub:`23`/*m*\ :sub:`12`
+* *M*\ :sub:`31` = *M*\ :sub:`32`\ *M*\ :sub:`21` −
+  (1 − *M*\ :sub:`23`\ *M*\ :sub:`32`)
+  *m*\ :sub:`12`/*m*\ :sub:`23`
+
+Multiple shortest geodesics
+---------------------------
+
+The shortest distance found by solving the inverse problem is
+(obviously) uniquely defined.  However, in a few special cases there are
+multiple azimuths which yield the same shortest distance.  Here is a
+catalog of those cases:
+
+* φ\ :sub:`1` = −φ\ :sub:`2` (with neither point at
+  a pole).  If α\ :sub:`1` = α\ :sub:`2`, the geodesic
+  is unique.  Otherwise there are two geodesics and the second one is
+  obtained by setting
+  [α\ :sub:`1`,α\ :sub:`2`] ←  [α\ :sub:`2`,α\ :sub:`1`],
+  [*M*\ :sub:`12`,\ *M*\ :sub:`21`] ← [*M*\ :sub:`21`,\ *M*\ :sub:`12`],
+  *S*\ :sub:`12` ← −\ *S*\ :sub:`12`.
+  (This occurs when the longitude difference is near ±180° for oblate
+  ellipsoids.)
+* λ\ :sub:`2` = λ\ :sub:`1` ± 180° (with
+  neither point at a pole).  If α\ :sub:`1` = 0° or
+  ±180°, the geodesic is unique.  Otherwise there are two
+  geodesics and the second one is obtained by setting
+  [α\ :sub:`1`,α\ :sub:`2`] ← [−α\ :sub:`1`,−α\ :sub:`2`],
+  *S*\ :sub:`12` ← −\ *S*\ :sub:`12`.  (This occurs when
+  φ\ :sub:`2` is near −φ\ :sub:`1` for prolate
+  ellipsoids.)
+* Points 1 and 2 at opposite poles.  There are infinitely many
+  geodesics which can be generated by setting
+  [α\ :sub:`1`,α\ :sub:`2`] ←
+  [α\ :sub:`1`,α\ :sub:`2`] + [δ,−δ], for arbitrary δ.
+  (For spheres, this prescription applies when points 1 and 2 are
+  antipodal.)
+* *s*\ :sub:`12` = 0 (coincident points).  There are infinitely many
+  geodesics which can be generated by setting
+  [α\ :sub:`1`,α\ :sub:`2`]_←
+  [α\ :sub:`1`,α\ :sub:`2`]_+ [δ,δ], for
+  arbitrary δ.
+
+Background
+----------
+
+The algorithms implemented by this package are given in Karney (2013)
+and are based on Bessel (1825) and Helmert (1880); the algorithm for
+areas is based on Danielsen (1989).  These improve on the work of
+Vincenty (1975) in the following respects:
+
+* The results are accurate to round-off for terrestrial ellipsoids (the
+  error in the distance is less then 15 nanometers, compared to 0.1 mm
+  for Vincenty).
+* The solution of the inverse problem is always found.  (Vincenty's
+  method fails to converge for nearly antipodal points.)
+* The routines calculate differential and integral properties of a
+  geodesic.  This allows, for example, the area of a geodesic polygon to
+  be computed.
+
+References
+----------
+
+* F. W. Bessel,
```

Comment:
It'd be nice to have these in `docs/references.rst` as well. To keep some sort of consistency across the documentation.

Reply:
OK, I'll move them over.